straxen/scada.py

```diff
@@ -22,12 +24,15 @@ def __init__(self, context=None):
             if you would like to query data via run_ids.
         """
         try:
-            self.SCData_URL = straxen.get_secret('SCData_URL')
-            self.SCLastValue_URL = straxen.get_secret('SCLastValue_URL')
-            self.SCADA_SECRETS = straxen.get_secret('SCADA_SECRETS')
+            self.SCData_URL = uconfig.get('scada', 'scdata_url')
```

Comment:
These values should be None right otherwise it's unclear to the user why the url does not work I image

Reply:
I'm not sure I follow. The configparser `get` is a bit different from the standard dictionary `get` -- is that what you're asking? This line will look for a section called 'scada' in the config, and variable named 'scdata_url'. If one doesn't exist it will throw an error.

Reply:
oops, I indeed was thinking too much about a dict operation. Thanks for the clarification!